### ISSUE TITLE
Block topic rows after maxAttempts

### ISSUE THREAD

**Issue by VladislavBakshanskij** [gruelbox/transaction-outbox#676]
In the current implementation of "ordered" tasks will not be blocked, but will be processed indefinitely. maybe it's worth blocking all tasks in a topic if the first task was not completed?

**Comment by badgerwithagun** [comment]
Yes, I've considered this for our purposes as well. I think it makes sense to be able to block a topic, but it will need to be a different configuration option to normal item-level blocking, and using a different configured threshold.
